10up/simple-local-avatars
# Local avatar delete-remove functionality added

### Description of the Change
From now on, users will see a new button to disassociate avatar rather than deleting permanently since there's a possibility of using the avatar image elsewhere. 


Closes #41 

### How to test the Change

- Login to a non-admin user account that has has no `upload_files` capability
- Go to profile edit page in WP dashboard
- Set a new avatar
- Click delete, it should delete file from the file system
- Set a new avatar again
- This time just remove, it should disassociate the image as avatar, however you should find the image still in the file system

### Changelog Entry

> Added - Avatar removal feature to disassociate images only instead of deleting from file syste,


### Credits

Props @jayedul 


### Checklist:
